Gemfile.lock

```diff
@@ -377,7 +377,7 @@ GEM
       net-ssh (>= 2.6.5, < 8.0.0)
     net-ssh (7.0.1)
     netrc (0.11.0)
-    nio4r (2.5.8)
+    nio4r (2.5.9)
```

Comment:
Why is this version bump required?

Reply:
I answered this in the PR description:
> A dependency of rails, nio4r failed to compile on installations having clang 16+. This has been patched upstream, so we bump up the version to make our setup script work.

To elaborate a bit more, the bundle step failed, because it resolves this dependency and tries to compile it, but fails. This was the issue: https://github.com/socketry/nio4r/issues/290
...and this was the patch: https://github.com/socketry/nio4r/commit/dd69a1c053bf13a580ec6a4e392c15c269239fd6